### PR TITLE
Update chart

### PR DESCRIPTION
**Pull request by dungdm93**
**What type of PR is this?**
/kind cleanup

### CODE REVIEW

File: helm/provisioner/templates/configmap.yaml [hunk 1]
```diff
@@ -33,20 +33,21 @@ data:
...
-       {{- if $classConfig.volumeMode }}
-       volumeMode: {{ $classConfig.volumeMode }}
-       {{- end }}
-       {{- if $classConfig.fsType }}
-       fsType: {{ $classConfig.fsType }}
-       {{- end }}
-       {{- if $classConfig.namePattern }}
-       namePattern: "{{ $classConfig.namePattern }}"
-       {{- end }}
+      hostDir:  {{ $classConfig.hostDir }}
```
Comment (dungdm93) [inline helm/provisioner/templates/configmap.yaml:36]: Indent 2 spaces

**Comment by cofyc** [comment]
You need to run `update-generated.sh` to update generated examples and check the output is what you expected.

**Comment by dungdm93** [comment]
Hello @cofyc 
```log
go: downloading golang.org/x/sys v0.0.0-20200610111108-226ff32320da
go: downloading gopkg.in/yaml.v2 v2.2.8
go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.3
Error: unable to find tester exec: "kubetest2-tester-exec" not found in PATH, could not locate "exec" deployer
make: *** [Makefile:54: e2e] Error 1
+ EXIT_VALUE=2
+ set +o xtrace
Cleaning up after docker in docker.
```
Above error from CI, I think there is some problems with `e2e-gke` test script. Could you please check it out.
Thanks.

**Comment by cofyc** [comment]
> Hello @cofyc
> 
> ```
> go: downloading golang.org/x/sys v0.0.0-20200610111108-226ff32320da
> go: downloading gopkg.in/yaml.v2 v2.2.8
> go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.3
> Error: unable to find tester exec: "kubetest2-tester-exec" not found in PATH, could not locate "exec" deployer
> make: *** [Makefile:54: e2e] Error 1
> + EXIT_VALUE=2
> + set +o xtrace
> Cleaning up after docker in docker.
> ```
> 
> Above error from CI, I think there is some problems with `e2e-gke` test script. Could you please check it out.
> Thanks.

this will be fixed in https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/210.

**Comment by dungdm93** [comment]
@cartyc still fails on `e2s-gke` test
```log
I0811 06:15:03.663840     698 commandutils.go:68] Checking existing of GCP ssh keys...
Deleting cluster e2e...
......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................done.
Deleted [https://container.googleapis.com/v1/projects/gke-up-c1-2-g1-4-up-clu-n/zones/us-central1-b/clusters/e2e].
ERROR: (gcloud.compute.networks.delete) Could not fetch resource:
 - The network resource 'projects/gke-up-c1-2-g1-4-up-clu-n/global/networks/default' is already being used by 'projects/gke-up-c1-2-g1-4-up-clu-n/global/firewalls/default-allow-internal'
Error: exit status 1
make: *** [Makefile:54: e2e] Error 1
+ EXIT_VALUE=2
```

File: helm/provisioner/templates/configmap.yaml (hunk 1 above)
Comment (cofyc) [inline helm/provisioner/templates/configmap.yaml:36]: s/hostDir:  {{/hostDir: {{/

remove one space between `:` and `{{`

Comment (dungdm93) [reply helm/provisioner/templates/configmap.yaml]: Updated

**Comment by cofyc** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**